bodhi/server/config.py

```diff
@@ -328,6 +328,9 @@ class BodhiConfig(dict):
         'bz_server': {
             'value': 'https://bugzilla.redhat.com/xmlrpc.cgi',
             'validator': six.text_type},
+        'cache_dir': {
+            'value': None,
+            'validator': _validate_none_or(_validate_path)},
```

Comment:
Can we also add this to ```production.ini``` so it's documented? ```production.ini``` actually automatically ends up in the Sphinx docs these days, so it's a handy way to make sure admin users know about it.